app/wmt-etl/poll-sqs.js

```diff
@@ -28,21 +23,19 @@ module.exports = function () {
           throw new Error('ETL already running')
         }
         return deleteSqsMessage(sqsClient, queueURL, data.Messages[0].ReceiptHandle).then(function () {
-          return listObjects(s3Client, S3.BUCKET_NAME).then(function (extracts) {
-            const excelExtracts = extracts.filter((extract) => extract.Key.endsWith('.xlsx'))
-            if (bothFilesPresent(excelExtracts)) {
-              return Promise.all(excelExtracts.map((extract) => getHasBeenRead(extract.Key)))
-                .then(function ([first, second]) {
-                  if (!first && !second) {
-                    return runEtl()
-                  } else {
-                    log.info('Files have been read')
-                    return 'Files have been read'
-                  }
-                })
+          return listObjects(s3Client, S3.BUCKET_NAME, S3.FILE_TO_PROCESS).then(function (extracts) {
+            if (extracts) {
+              return getHasBeenRead(S3.FILE_TO_PROCESS).then(function (fileHasBeenRead) {
+                if (!fileHasBeenRead) {
+                  return runEtl()
+                } else {
+                  log.info('File has been read')
+                  return 'File has been read'
+                }
+              })
             }
-            log.info('only one file updated')
-            return 'Only one file updated'
+            log.info(`file ${S3.FILE_TO_PROCESS} does not exist in bucket`)
+            return `file ${S3.FILE_TO_PROCESS} does not exist in bucket`
```

Comment:
This would happen in the case where something other than this file has been put into the bucket?
Maybe we should put that in the error message?

"no unprocessed extracts named ${S3.FILE_TO_PROCESS} found in bucket"